tasks/templates/tasks/session_confirm_delete.html

```diff
@@ -15,11 +15,10 @@ <h2>{% trans "Sessie verwijderen" %}</h2>
                 <p>
                     Weet u zeker dat u deze sessie <strong>{{ session.order }}</strong> in de aanvraag <em>{{ session.study.proposal.title }}</em> wilt verwijderen?
                 </p>
-                <input class="pure-button pure-button-primary"
+                <input class="button btn-danger"
```

Comment:
Wait, this works? Huh, how....

Reply:
ehm ... It turns red when hovering, which I could not explain, but kindoff liked ...